### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://translate.mattermost.com/projects/mattermost/mattermost-server_master/) for mattermost-languages-shipped/mattermost-server.

Current translation status:

Weblate translation status

```release-note
Adding Bulgarian(bg) and Swedish(sv) to mattermost-server
```

### CODE REVIEW

File: i18n/bg.json [hunk 1]
```diff
@@ -0,0 +1,8397 @@
...
+    "id": "api.templates.invite_subject",
+    "translation": "[{{.SiteName}}] {{.SenderName}} ви покани да се присъедините към екипа {{.TeamDisplayName}}"
+  },
+  {
+    "id": "api.templates.invite_guest_subject",
+    "translation": "[{{.SiteName}}] {{.SenderName}} ви покани да се присъедините към екипа {{.TeamDisplayName}} като гост"
+  },
+  {
+    "id": "api.templates.invite_body_guest.info",
+    "translation": "[[{{.SenderName}}]], ви покани да се присъедините към екипа [[{{.TeamDisplayName}}] като гост."
```
Comment (metanerd) [inline i18n/bg.json:4444]:
```suggestion
    "translation": "[[{{.SenderName}}]], ви покани да се присъедините към екипа [[{{.TeamDisplayName}}]] като гост."
```

File: i18n/bg.json [hunk 2]
```diff
@@ -0,0 +1,8397 @@
...
...ни, включително следното:\n\n{{.BotNames}}{{end}}Можете да вземете собственост върху всеки бот, като го активирате на **Интеграции> Бот профили** и създадете на нови маркери за бота.\n\nЗа повече информация вижте нашата [документация](https://docs.mattermost.com/developer/bot-accounts.html#what-happens-when-a-user-who-owns-bot-accounts-is-disabled).{{else}}{{if .printAllBots}}{{.UserName}} са деактивирани. Те управляваха следните бот акаунти, които все още са активирани.\n\n{{.BotNames}}\n{{else}}{{.UserName}} е деактивиран. Те управляваха {{.NumBots}} бот профили, които все още са активирани, включително следното:\n\n{{.BotNames}}{{end}}Силно ви препоръчваме да поемете собственост върху всеки бот, като го активирате отново на **Интеграции> Бот профили** и създадете на нови маркери за бота.\n\nЗа повече информация вижте нашата [документация](https://docs.mattermost.com/developer/bot-accounts.html#what-happens-when-a-user-who-owns-bot-accounts-is-disabled).\n\nАко искате бот профили да се деактивират автоматично след деактивиране на собственика, задайте „Деактивиране на бот профили, когато собственикът е деактивиран“ в **Системна конзола > Интеграции > Бот профили** на true.{{end}}"
```
Comment (metanerd) [inline i18n/bg.json:5258]: LGTM.

File: i18n/sv.json [hunk 3]
```diff
@@ -0,0 +1,8401 @@
...
+    "translation": {
+      "one": "[{{.SiteName}}] Ny notis den {{.Day}} {{.Month}} {{.Year}}",
+      "other": "[{{.SiteName}}] Nya notiser den {{.Day}} {{.Month}} {{.Year}}"
+    }
+  },
+  {
+    "id": "api.email_batching.send_batched_email_notification.body_text",
+    "translation": {
+      "one": "Du har en ny notifikation.",
+
```
Comment (metanerd) [inline i18n/sv.json:7142]:
```suggestion
    "translation": "Kunde inte tysta kanalen {{.Channel}} eftersom du inte är medlem."
```

Comment (metanerd) [inline i18n/sv.json:7453]:
```suggestion
    "translation": "Kanalens namn får inte vara längre än {{.Length}} tecken."
```

**Comment by metanerd** [comment]
```release-note
Adding Bulgarian(bg) and Swedish(sv) to mattermost-server. 
```

**Review by metanerd — APPROVED** [review]
LGTM

**Review by jasonblais — APPROVED** [review]
:tada:

**Pull request merged**